lib/Image/Operation/Resize.php

```diff
@@ -17,27 +17,51 @@
  */
 class Resize extends ImageOperation {
 
-   private $w, $h, $crop;
+	private $parameters_or_w, $h, $crop;
```

Comment:
Can we list properties separately, with a DocBlock for each parameter?

Reply:
they are seperate now. Probably the doctype part will need some polishing thought.

Reply:
About the quality param - I'll skip it for now. Let's finish this one and than we can start adding new filters.